docs/chapters/architecture/SWC/software-container.rst

```diff
@@ -0,0 +1,358 @@
+:orphan:
+
+.. _software-container:
+
+
+Software Container
+==================
+
+Introduction and goals
+----------------------
+
+The Software Container is a framework that manage and contain applications created
+and vetted by third party developers in an automotive setting. A launcher 
+(a UI for example) sends a signal to the Software Container to start a new 
+Container. It can then configure the Container using a documented JSON format and 
+can launch process inside the Container. This can then be used to contain 
+applications from each other and make sure they are not interfering with each 
+other and to limit/manage resources available to the applications.
+
+Each container is configured using gateways, where each gateway has a scope of 
+influence (network, dbus, files, pulseaudio, for example). The gateways get a 
+configuration snippet from the Launcher which they will enforce.
+
+Requirements overview
+^^^^^^^^^^^^^^^^^^^^^
+
+Basic usage
+"""""""""""
+
+- Software Container Agent component is started at the PELUX boot-up
```

Comment:
```suggestion
- The SoftwareContainerAgent component is started at the PELUX boot-up
```

Reply:
Possibly and in that case also change the following uses of Software Container Agent to use this way of writing.